src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptionsUpdateHandler.cs

```diff
@@ -15,7 +15,7 @@ namespace System.Text.Json
     internal static class JsonSerializerOptionsUpdateHandler
     {
         [RequiresDynamicCode(JsonSerializer.SerializationRequiresDynamicCodeMessage)]
-        public static void ClearCache(Type[]? types)
+        public static void ClearCache()
```

Comment:
This is the signature the update handler is expected to have.  It's _possible_ we've made the hot reload invocation code more robust and able to invoke a parameterless method as well, but you'll want to triple-check that.

Reply:
ok, reverted